junit5-api/src/main/java/org/junit/gen5/api/extension/ParameterResolver.java

```diff
@@ -32,61 +31,46 @@
  * <p>Implementations must provide a no-args constructor.
  *
  * @since 5.0
- * @see #supports(Parameter, Optional, ExtensionContext)
- * @see #resolve(Parameter, Optional, ExtensionContext)
+ * @see #supports(ParameterContext, ExtensionContext)
+ * @see #resolve(ParameterContext, ExtensionContext)
+ * @see ParameterContext
  */
 @API(Experimental)
 public interface ParameterResolver extends Extension {
 
 	/**
-	 * Determine if this resolver supports resolution of the given {@link Parameter}
-	 * for the supplied {@code target} and {@link ExtensionContext}.
+	 * Determine if this resolver supports resolution of the given
+	 * {@link ParameterContext} in the supplied {@link ExtensionContext}.
```

Comment:
I would use wording in line with the original text.

A `ParameterResolver` is still intended to resolve parameters, not parameter contexts.

So that would lead to wording like this:

> Determine if this resolver supports resolution of the Parameter in the supplied ParameterContext for the supplied ExtensionContext.

The same applies to the wording of the `resolve()` method.